### ISSUE TITLE
Implement The Hide Complete Screen To Pause Menu Too

### ISSUE THREAD

**Issue by NewAlexGamer** [absolllute/Mega-Hack-Pro-Future#914]
tl;dr -> currently you can hide the complete screen to see the level end using a simple arrow located in the top middle of the screen. why not extend this to the pause menu too instead of having a separate hack for it (hide pause menu)

actual motivation for this to be implemented:
this is just to make things more efficient. why did absolute not make another hack for the complete screen too, why this method specifically and not the same hack for both? "well with the pause menu you can hide it and take screenshots", fine, but why would you want a screenshot of a part of level, possibly not even polished and maybe not even meant to be seen in it's fine details, and not a screenshot of the entire end screen where it was made specifically to be shown and make jaws drop (remember, the arrow is in the way)

**Comment by NewAlexGamer** [comment]
reading this again seems more like i'm trying to make people think if it's better for both to have either the hack or the arrow
